packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -566,14 +567,20 @@ export function flushRoot(root: FiberRoot, expirationTime: ExpirationTime) {
   flushSyncCallbackQueue();
 }
 
-export function flushInteractiveUpdates() {
-  if (workPhase === RenderPhase || workPhase === CommitPhase) {
-    // Can't synchronously flush interactive updates if React is already
-    // working. This is currently a no-op.
-    // TODO: Should we fire a warning? This happens if you synchronously invoke
-    // an input event inside an effect, like with `element.click()`.
+export function flushDiscreteUpdates() {
+  if (workPhase === CommitPhase || workPhase === BatchedPhase) {
```

Comment:
Why is this a no-op in `batchedUpdates`? Is there a test that failed otherwise?

Reply:
Yep, a bunch of tests failed from memory.

Reply:
Can you put a TODO there? I'll take a look later. Doesn't have to block merge.

Reply:
Added a TODO. Let's resync with this once I'm back from PTO; this at least unblocks us internally.